lib/volcanic/authenticator.rb

```diff
@@ -20,6 +23,15 @@ module Base
       def config
         Volcanic::Authenticator::V1::Config
       end
+
```

Comment:
consider making this part of the `config` method.

```
def config
  yield Volcanic::Authenticator::V1::Config if block_given?
  Volcanic::Authenticator::V1::Config
end
```

Reply:
noted